shared/chat/conversation/normal/index.desktop.js

```diff
@@ -81,6 +82,21 @@ class Conversation extends React.PureComponent<Props, State> {
     const fileList = e.dataTransfer.files
     const paths = fileList.length ? Array.prototype.map.call(fileList, f => f.path) : []
     if (paths) {
+      for (let path of paths) {
```

Comment:
`if (paths)` will always be true, right? `if (paths.length)` I guess?

Reply:
:+1: